app/scripts/lib/rpc-method-middleware/handlers/request-accounts.js

```diff
@@ -85,6 +85,7 @@ async function requestEthereumAccountsHandler(
 
   try {
     const caip25Approval = await requestCaip25ApprovalForOrigin();
+    console.log({ caip25Approval });
```

Comment:
These should be stripped before merge

Reply:
thanks for catching this https://github.com/MetaMask/metamask-extension/pull/27782/commits/cf15677593a0376aedb2f4212faa0e4f0bdf1b34

Reply:
These were reintroduced

Reply:
i don't see this change anymore. Should be gone now if you saw it come back